### ISSUE TITLE
`slaveoutput` and `slaveinput` are now called `workeroutput` and `workerinput` in pytest-xdist

### ISSUE THREAD

**Issue by martinmaillard** [pytest-dev/pytest-cov#191]
In the latest version of pytest-xdist (1.22.1), the "master/slave" terminology has been replaced by "master/worker" (https://github.com/pytest-dev/pytest-xdist/pull/268).

I noticed the issue because I kept getting the following warning:
> The following slaves failed to return coverage data, ensure that pytest-cov is installed on these slaves.

They provide aliases for backward compatibility but they don't seem to be available everywhere...

**Comment by nicoddemus** [comment]
Hi @martinmaillard, I'm not sure what the issue is exactly... it seems pytest-cov is emitting a warning using `slave` terminology, but that should not be a problem. Can you describe your issue in more details please? Thanks!

**Comment by martinmaillard** [comment]
Using pytest-xdist==1.22.1 and pytest-cov==2.5.1, if I run:

`pytest --cov . -n 2`

I always get the following error message:
```
[...]
---------------------- coverage: failed slaves -----------------------
The following slaves failed to return coverage data, ensure that pytest-cov is installed on these slaves.
gw0
gw1
```

From what I can tell, the results are not actually affected.

I tried to track down the issue and I found out that in [`DistMaster.testnodedown`](https://github.com/pytest-dev/pytest-cov/blob/master/src/pytest_cov/engine.py#L214), the `node` object doesn't have a `slaveoutput` property. It does have a `workeroutput` property though. So I'm guessing the "aliases for backward compatibilty" of pytest-xdist are not always available...

**Comment by nicoddemus** [comment]
Oh then it is a bug in `pytest-xdist`, the alias should be available:

https://github.com/pytest-dev/pytest-xdist/blob/63b329c251fb1bca23ad38a5184eaba594fb7b12/xdist/workermanage.py#L308-L311

This line:

```python
self.workeroutput = kwargs['workeroutput']
```

Should have been:

```python
self.workeroutput = self.slaveoutput = kwargs['workeroutput']
```

It is a shame we missed that during review. 😞

**Comment by nicoddemus** [comment]
pytest-dev/pytest-xdist#286 opened with a fix.

**Comment by nicoddemus** [comment]
This fix will go out in pytest-dev/pytest-xdist#287.

**Comment by martinmaillard** [comment]
Wow that was fast! Thank you :)
